### PR TITLE
Update Kured chart to create Service for prometheus metrics

### PR DESCRIPTION
**Pull request by razaj92**
Signed-off-by: Raza Jhaveri <razajhaveri@googlemail.com>

#### What this PR does / why we need it:
Creates a svc to expose the prometheus exporter 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ x ] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [ x ] Chart Version bumped
- [ x ] Variables are documented in the README.md

### CODE REVIEW

File: stable/kured/templates/daemonset.yaml [hunk 1]
```diff
@@ -58,6 +58,9 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+          ports:
+            - name: http
+              containerPort: {{ .Values.service.port }}
```
Comment (lachie83) [inline stable/kured/templates/daemonset.yaml:63]: Should the service.port and the containerPort always be the same thing? Is there a way to tell kured to run on a different port?

**Review by lachie83 — CHANGES_REQUESTED** [review]

File: stable/kured/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  labels:
+    app: {{ template "kured.name" . }}
+    chart: {{ template "kured.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+  - name: http
+    port: {{ default "8080" .Values.service.port }}
```
Comment (davidkarlsen) [inline stable/kured/templates/service.yaml:16]:
```suggestion
    port: {{ .Values.service.port }}
```
which is according to best practices

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**